src/components/Visualizations/GaugeChart.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Styled = styled.div` 
+   font-size: 60%;
+   padding: 5px;
+   margin-right: 15px;
+   position: relative;
+   
+   :last-child {
+    margin-right: 0 !important;
+   }
+
+  .gauge {
+    display:inline-block;
+    position:relative;
+    width:10em;
+    height:5em;
+    overflow:hidden;
+    
+  }
+  
+  .gauge:before, .gauge:after, .meter {
+    position:absolute;
+    display:block;
+    content:"";
+  }
+  
+  .gauge:before, .meter { width:10em; height:5em; }
+  .gauge:before { border-radius:5em 5em 0 0; background:#2a2a2a; }
+  
+  .gauge:after {
+    position:absolute;
+    bottom:0;
+    left:2.5em;
+    width:5em;
+    height:2.5em;
+    background:rgb(33, 34, 44);
+    border-radius:2.5em 2.5em 0 0;
+  }
+  
+  .meter {
+    top:100%;
+    transform-origin:center top;
+    border-radius:0 0 6em 6em;
+    transform:rotate(${props => props.percent}turn);
+  }
+  
+  .percentage .meter { background:${props => props.meterColor}; }
+  .percentage-container {
+    position:absolute;
+    bottom:-.75em;
+    left:2.5em;
+    z-index:1000;
+    width:5em;
+    height:2.5em;
+    overflow:hidden;
+  }
+  
+  .percentage-indicator {
+    font:bold 1.25em/1.6 sans-serif;
+    color:${props => props.meterColor};
+    
+    white-space:pre;
+    vertical-align:baseline;
+    user-select:none;
+    text-align: center;
+  }
+
+  .caption {
+      position: relative;
+      text-align: center;
+      left: 4px;
+      color: rgb(179,179,179);
+      font-size: 12px;
+      bottom: 5px;
+      text-transform: uppercase;
+      max-width: 90px;
+      max-height: 16px;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+  }
+
+  .win {
+    position:absolute;
+    left:0.9em;
+    bottom: 0.7em;
+    color: #080808;  
+  }
+
+  .loss {
+    position:absolute;
+    right:0.8em;
+    bottom: 0.7em;
+    color: rgb(144, 144, 144);   
+  }
+`;
+const computeMeterPercent = value => 0.005 * value;
+
+const computeMeterColor = (value) => {
+  if (value < 45) {
+    return 'rgb(179,132,91)';
+  } else if (value < 50) {
+    return 'rgb(156,148,96)';
+  } else if (value < 53) {
+    return 'rgb(140,159,99)';
+  }
+  return 'rgb(117,176,103)';
+};
+
+const GaugeChart = ({ value, caption }) => (
+  <Styled percent={computeMeterPercent(value)} meterColor={computeMeterColor(value)}>
+    <div className="caption">{caption}</div>
```

Comment:
I think we could display the number of matches as well, maybe as parens after the category, like `DIRE (1020)`